fhir_transformer/src/main.py

```diff
@@ -24,42 +25,43 @@ def process_event(msg):
     # Do stuff
     msg_value = json.loads(msg.value())
     msg_topic = msg.topic()
-    logging.info("Transformer")
-    logging.info(msg_topic)
-    logging.info(msg_value)
+    logger.info("Transformer")
+    logger.info(msg_topic)
+    logger.info(msg_value)
 
     try:
-        record = {"example": {"patient": "fhirised"}}
+        logger.info(msg_value["analysis"])
```

Comment:
Maybe remove this logging before merging

Reply:
Nice catch!